### PR TITLE
feat(config): update ZDB5100 config to expand on parameter 1

### PR DESCRIPTION
**Pull request by LordMike**

### CODE REVIEW

**Comment by LordMike** [comment]
@zwave-js-bot semantic?

**Comment by AlCalzone** [comment]
Why the hell is this PR failing? It works locally. Other PRs build fine.

**Comment by AlCalzone** [comment]
@zwave-js-bot rename commit

**Comment by AlCalzone** [comment]
Okay, there was an error getting swallowed...

File: packages/config/config/devices/0x0234/zdb5100.json [hunk 1]
```diff
@@ -17,17 +17,50 @@
 	},
 	"supportsZWavePlus": true,
 	"paramInformation": {
-		"1": {
-			"label": "Operating pushbutton(s) for dimmer",
-			"description": "This parameter specifies which pushbutton(s) that controls the built-in dimmer.",
+		"1[0x01]": {
+			"label": "Pusbutton 1 controls the dimmer",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0234/zdb5100.json]:
```diff
- Pusbutton
+ Pushbutton
```
(same for the other params)

File: packages/config/config/devices/0x0234/zdb5100.json [hunk 2]
```diff
@@ -17,17 +17,50 @@
...
+			"description": "Enabling this parameter, lets pushbutton 1 control the built-in dimmer.",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0234/zdb5100.json]: I think the comma behind `parameter` does not belong there. (same for the other params)

**Review by AlCalzone — CHANGES_REQUESTED** [review]

**Pull request merged**